bindings/rust/s2n-tls-sys/templates/Cargo.template

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "s2n-tls-sys"
 description = "A C99 implementation of the TLS/SSL protocols"
-version = "0.1.2"
+version = "0.2.0"
```

Comment:
why are we minor bumping here?

Reply:
You are correct, was interpreting some commits as minor changes.